homeworks/nazar_kostiv/pyats01_hw.py

```diff
@@ -52,4 +52,4 @@ def test_divide(self, steps, num1, num2):
     parser.add_argument('-num1', dest='num1', type=int, required=False, default=parameters['num1'])
     parser.add_argument('-num2', dest='num2', type=int, required=False, default=parameters['num2'])
     args, sys.argv[1:] = parser.parse_known_args(sys.argv[1:])
-    main(num1=args.num1, num2=args.num2)
+    main(num1=args.num1, num2=args.num2)
```

Comment:
@jsakhno an empty line is required at the end of the file.

Reply:
@extsoft fixed